src/app/modules/installers/installers.tpl.jade

```diff
@@ -21,7 +21,7 @@ mixin link(href)
 				+link('#/installers/old') here
 				| , and Linux Installation Instructions are
 				+link('#/installers/linux') here.
-			iframe.download-version-frame(src='http://bloomlibrary.org/channels/DownloadPointers_Beta.htm')
+			iframe.download-version-frame(src='http://bloomlibrary.org/channels/DownloadPointers_Beta.htm', height='30px', width='100%', frameborder='0', scrolling='no')
```

Comment:
I just copied this from the release version (a few lines up in the same file).  But you're right that it would be better to not have height and width there.  The frameborder and scrolling attributes have no css equivalent, though.